packages/server/postgres/migrations/1651532750081_domainIndex.ts

```diff
@@ -0,0 +1,18 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  CREATE INDEX IF NOT EXISTS "idx_OrganizationApprovedDomain_domain" ON "OrganizationApprovedDomain"("domain");`)
```

Comment:
by indexing on the domain filed, we can get rid of the rethinkdb SecureDomain table